redbeams/src/main/java/com/sequenceiq/redbeams/repository/DatabaseServerConfigRepository.java

```diff
@@ -31,21 +31,11 @@
 
     Optional<DatabaseServerConfig> findByName(String name);
 
-    @Query("SELECT s FROM DatabaseServerConfig s WHERE s.workspaceId = :workspaceId AND s.environmentId = :environmentId "
-            + "AND (s.name IN :names OR s.resourceCrn IN :names)")
-    Set<DatabaseServerConfig> findByNameInAndWorkspaceIdAndEnvironmentId(
-            @Param("names") Set<String> names,
-            @Param("workspaceId") Long workspaceId,
-            @Param("environmentId") String environmentId);
-
-    @Query("SELECT c.resourceCrn FROM DatabaseServerConfig c WHERE c.accountId = :accountId")
-    List<Crn> findAllResourceCrnsByAccountId(@Param("accountId") String accountId);
-
     @Query("SELECT c.resourceCrn FROM DatabaseServerConfig c WHERE c.name = :name")
     Optional<Crn> findResourceCrnByName(@Param("name") String name);
 
     @Query("SELECT c.name as name, c.resourceCrn as crn FROM DatabaseServerConfig c WHERE c.resourceCrn IN (:resourceCrns)")
```

Comment:
Remark: an idea for simplification of the query: the projection and the query can both be done via spring method naming, by:
- renaming method to findByResourceCrnIn(), and 
- renaming getCrn() to getResourceCrn() in the projection.
Just if you wanted to simplify, if you have no time, no problem.

Reply:
good point! I changed it, tested it, still looks good:
```
curl -H 'Content-Type: application/json' -H "x-cdp-actor-crn: crn:altus:iam:us-west-1:cloudera:user:adam.horvath@cloudera.com" http://localhost:8087/redbeams/api/v4/databaseservers/crn:cdp:redbeams:us-west-1:cloudera:databaseServer:8aeb2ddc-e4fb-4bb3-80ba-5a2e665f60bb
{"message":"Doesn't have 'environments/describeDatabaseServer' right on databaseServer [crn: crn:cdp:redbeams:us-west-1:cloudera:databaseServer:8aeb2ddc-e4fb-4bb3-80ba-5a2e665f60bb]."}
```